Jenkinsfile

```diff
@@ -93,7 +93,7 @@ pipeline {
       archiveArtifacts artifacts: 'datagov-deploy.tar.gz', onlyIfSuccessful: true
```

Comment:
I don't know if this should be changed or how it's generated.

Reply:
I limited my changes only to places that reference the URL. I don't believe we need to change this one.